source/configure/plugins-configuration-settings.rst

```diff
@@ -808,9 +808,134 @@ Transcriber model size
+| **Note**: The call transcriber threads setting will affect the performance of the job service. Refer to the `configure call recordings and transcriptions <calls-deployment.html#configure-recording-transcriptions-and-live-captions>`__ documentation for more information. This setting is available starting in plugin version 0.26.2.                                                                                                                                                            |
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+.. config:setting:: plugins-enablelivecaptions
+  :displayname: Enable live captions (Experimental) (Plugins - Calls)
+  :systemconsole: Plugins > Calls
+  :configjson: PluginSettings.Plugins.com.mattermost.calls.enablelivecaptions
+  :environment: N/A
+  :description: Enables live captioning of calls.
+
+Enable live captions (Experimental)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../_static/badges/ent-selfhosted-only.rst
+  :start-after: :nosearch:
+
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------+
+| - **true**: Enables live captioning of calls.                                                                                                                                  | - System Config path: **Plugins > Calls**                                                      |
+| - **false**: **(Default)** Live captions functionality is disabled.                                                                                                            | - ``config.json`` setting: ``PluginSettings.Plugins.com.mattermost.calls.enablelivecaptions``  |
+|                                                                                                                                                                                |                                                                                                |
+| Live captions are generated from the call participants' audio tracks and the resulting captions can be optionally displayed on the call clients by clicking the `[cc]` button. |                                                                                                |
```

Comment:
Note to self: this will need updating once we ship the call settings as the button is moved to a menu item.